packages/flutter/lib/src/material/text_selection.dart

```diff
@@ -104,8 +104,7 @@ class _TextSelectionToolbarLayout extends SingleChildLayoutDelegate {
   }
 }
 
-/// Draws a single text selection handle. The [type] determines where the handle
-/// points (e.g. the [left] handle points up and to the right).
+/// Draws a single text selection handle which points up and to the left.
```

Comment:
The old comment seemed to explain this better? Has the behavior here changed?

Reply:
The behavior hasn't actually changed, but the old comment refers to [type] and [left] which aren't actually used in this class. This class always draws the caret up and to the right. It then gets rotated later in `buildHandle`, at `material/text_selection.dart:173`.

Reply:
Ahh ok, I see!